### PR TITLE
update eslint versions + lint src test + stage in travis

### PR DESCRIPTION
**Pull request by DavidFaizulaev**
* Updated eslint packages
* Ran eslint on src/ & test/ directories
* Added eslint stage in travis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "types-check": "tsc -p .",
     "test": "mocha ./test/unit-test/*-test.js ./test/**/**/*-test.js --require ts-node/register test/integration-tests/nest-js/*.spec.ts",
     "integration-tests": "nyc mocha ./test/**/**/*-test.js --require ts-node/register test/integration-tests/nest-js/*.spec.ts",
-    "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls"
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls",
+    "lint": "node_modules/eslint/bin/eslint.js src/**"
```
Comment (kobik) [inline package.json]: `node_modules/eslint/bin/eslint.js src/**/*.js test/**/*.js`

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+src/index.d.ts
```
Comment (kobik) [inline .eslintignore]: no need for this as we'll use `node_modules/eslint/bin/eslint.js src/**/*.js test/**/*.js` in package.json

**Review by kobik — CHANGES_REQUESTED** [review]
@DavidFaizulaev see comments

File: package.json (hunk 1 above)
Comment (DavidFaizulaev) [reply package.json]: done

File: .eslintignore (hunk 2 above)
Comment (DavidFaizulaev) [reply .eslintignore]: deleted file

**Review by kobik — APPROVED** [review]

**Pull request merged**